deploy/src/loadEnv.js

```diff
@@ -111,10 +147,8 @@ if (BRIDGE_MODE === 'NATIVE_TO_ERC') {
     BRIDGEABLE_TOKEN_NAME: envalid.str(),
     BRIDGEABLE_TOKEN_SYMBOL: envalid.str(),
     BRIDGEABLE_TOKEN_DECIMALS: envalid.num(),
-    FOREIGN_DAILY_LIMIT: bigNumValidator(),
     FOREIGN_MIN_AMOUNT_PER_TX: bigNumValidator(),
-    DEPLOY_REWARDABLE_TOKEN: envalid.bool(),
-    BLOCK_REWARD_ADDRESS: addressValidator()
+    DEPLOY_REWARDABLE_TOKEN: envalid.bool()
```

Comment:
So, we don't need `BLOCK_REWARD_ADDRESS` in this mode, do we?

Reply:
Sorry on the confusion, We actually need `BLOCK_REWARD_ADDRESS` for foreign side when `DEPLOY_REWARDABLE_TOKEN` is `true` on `native_to_erc`. Fixed here https://github.com/poanetwork/poa-bridge-contracts/pull/228/commits/857c860dcfbd3f3ab5d3181a108cd7fcadff0357